src/frontend/planner_test/src/lib.rs

```diff
@@ -253,6 +254,12 @@ pub struct TestCaseResult {
     pub with_config_map: Option<BTreeMap<String, String>>,
 }
 
+macro_rules! report {
+    ($error:expr) => {
+        format!("{:#}", $error.as_report())
```

Comment:
Maybe we can add helper methods in `AsReport`: `to_report_string()`, `to_report_string_pretty()`..?